gatsby-node.js

```diff
@@ -10,12 +10,15 @@ var path = require("path")
 
 exports.createPages = ({ actions, graphql }) => {
   const { createPage } = actions
-  const applyTemplate = path.resolve(`src/templates/apply.js`)
+  const applyTemplate = path.resolve(`src/templates/role.js`)
   return graphql(`
     {
       allMarkdownRemark(filter: { fileAbsolutePath: { regex: "/apply/" } }) {
         edges {
           node {
+            fields {
+              slug
```

Comment:
used this so that we can access the page path for our roles from graphql on other pages